src/prefect/tasks/gcp/bigquery.py

```diff
@@ -387,8 +387,9 @@ def run(
                 job_config=job_config,
             ).result()  # block until job is finished
         except Exception as exception:
-            for error in load_job.errors:
-                self.logger(error)
+            for error in exception.errors:
+                self.logger.error(error)
```

Comment:
After initializing `load_job` correctly (see above), `load_job.errors` might actually contain some useful information. See thread here https://github.com/googleapis/google-cloud-python/issues/6573.

Something like the following might work
```python
except Exception as exception:
            self.logger.error(exception)
            try:
                for error in load_job.errors:
                    self.logger.exception(error)
            except NameError:
                pass
            raise FAIL(exception) from exception
```

Reply:
Ah good suggestion! I didn't actually dive in to which errors[] the message was referring. But I opted to log everything :)